pkg/sdn/plugin/sdn-cni-plugin/openshift-sdn.go

```diff
@@ -0,0 +1,511 @@
+// +build linux
```

Comment:
The directory name ends up being the Go binary name, which is why I went more descriptive.  I don't like it either, but I felt like 'sdn' or 'cni' were too generic.  I thought about just "openshift-sdn" which is what its installed name is, but that would conflict in the future with a possible standalone openshift-sdn node process.  Suggestions appreciated...